src/include/dng/inheritance_model.h

```diff
@@ -54,6 +49,25 @@ class InheritanceModel{
 
     InheritancePattern GetInheritancePattern();
 
+
+    const std::pair<std::string, InheritancePattern> INHERITANCE_KEYS[13] = {
+        {"DEFAULT", InheritancePattern::AUTOSOMAL},
```

Comment:
Probably don't need to support "DEFAULT" as a key, but if we do, it should be linked to DEFAULT, not AUTOSOMAL.